repository/interface.go

```diff
@@ -0,0 +1,13 @@
+package repository
```

Comment:
rename file to repository.go

Reply:
right, and we can add the documentation (package overview) to this file, I made the same change with `services/interface.go`